awscli/clidocs.py

```diff
@@ -186,18 +190,47 @@ def doc_relateditem(self, help_command, related_item, **kwargs):
         )
         doc.write('\n')
 
-    def _document_enums(self, argument, doc):
+    def _document_enums(self, model, doc):
         """Documents top-level parameter enums"""
-        if hasattr(argument, 'argument_model'):
-            model = argument.argument_model
-            if isinstance(model, StringShape):
-                if model.enum:
-                    doc.style.new_paragraph()
-                    doc.write('Possible values:')
-                    doc.style.start_ul()
-                    for enum in model.enum:
-                        doc.style.li('``%s``' % enum)
-                    doc.style.end_ul()
+        if isinstance(model, StringShape):
```

Comment:
It probably makes sense to document enums as well when they are in nested structures. Right now it looks like we do not account for that.

Reply:
In all the cases I've seen enums were already documented in `.documentation` so it will be duplication